test/src/Main.purs

```diff
@@ -123,8 +208,15 @@ timeTest = do
   log $ "\nUNFORMATDATETIME"
   DT.traceAnyA $ FDT.unformatDateTime "YYYY-DD-MM SSS" "3456-09-10 333"
 
-
-main ∷ forall e. Eff (console ∷ CONSOLE | e) Unit
-main = do
-  timeTest
-  numeral
+main :: forall e.
+  Eff ( process :: PROCESS, exception :: EXCEPTION, console :: CONSOLE | e)
+    (Canceler ( process :: PROCESS, exception :: EXCEPTION, console :: CONSOLE | e))
+main = execTests tests true
+  where
+  tests = do
+    log "Testing time functions..."
+    timeTest
+    passed <- get
+    when (passed /= true) (throwError (error "Tests did not pass."))
+    --numeralTests
+    --formattingTests
```

Comment:
Are these intentionally commented out?

Sorry about the repeated back-and-forth on this issue :)